settings/pkg/service/v0/settings.go

```diff
@@ -476,6 +475,18 @@ func defaultRoleAssignments() []*settingsmsg.UserRoleAssignment {
 		}, {
 			AccountUuid: "932b4540-8d16-481e-8ef4-588e4b6b151c",
 			RoleId:      BundleUUIDRoleUser,
+		}, {
+			AccountUuid: "534bb038-6f9d-4093-946f-133be61fa4e7",
+			RoleId:      BundleUUIDRoleUser,
+		},
+		// kjohnson with additional role "space-manager" (ListAllSpaces + CreateSpace)
+		{
+			AccountUuid: "534bb038-6f9d-4093-946f-133be61fa4e7",
+			RoleId:      ListAllSpacesPermissionID,
```

Comment:
You're assigning a permission id as a role id. I guess you need to create a `SpaceManagers` role and use the role id here.

Reply:
thanks for that hint!